src/test/resources/org/prebid/server/it/openrtb2/visx/test-auction-visx-request.json

```diff
@@ -0,0 +1,70 @@
+{
+  "id": "tid",
+  "imp": [
+    {
+      "id": "test-imp-id",
+      "banner": {
+        "format": [
+          {
+            "w": 300,
+            "h": 250
+          }
+        ]
+      },
+      "secure": 1,
+      "ext": {
+        "visx": {
+          "ga_parms": {
+            "place_on_page": "ATF"
```

Comment:
I know it's an example they use in Go, but unlike us - then didn't introduce ExtImpVisx model, yet have some fields description in `visx.json` schema... 

We'll ask @rpanchyk whether we should keep the ExtImp model or skip it (will have to re-write the bidder class, but it's the easiest one).

Reply:
Okay, nevermind this one, we'll leave it as it is.